### PR TITLE
[MARVP-29] Implement Event location page

### PR DESCRIPTION
**Pull request by krishk95**
Related to : https://konysolutions.atlassian.net/browse/MARVP-29

### CODE REVIEW

File: KonyAccelerate/controllers/mobile/frmEventLocationController/frmEventLocationController.js [hunk 1]
```diff
@@ -1,5 +1,86 @@
...
+define({
+  
+    /**
+     * @function formPreshowAction
+     * @descripiton This function is invoked in the pre show of the form which is used to fetch the event data
+     * @private
+     */
+    formPreshowAction: function() {
+        let eventData = kony.store.getItem("accelerateEventData");
+      	if(eventData !== null && eventData !== undefined){
```
Comment (vijaysamanthapudi) [inline KonyAccelerate/controllers/mobile/frmEventLocationController/frmEventLocationController.js]: try to use kony.sdk.isNullOrUndefined generic function for NULL/ undefined checks

File: KonyAccelerate/controllers/mobile/frmLoginController/frmLoginController.js [hunk 2]
```diff
@@ -1,6 +1,74 @@
...
+        kony.store.setItem("accelerateEventData",null);
+    },
+    /**
+     *	@function formatInnerLocationData
+     * 	@description This function is used to format the inner location data as required by the application (JSON Format)
+     *	@param eventInnerLocationData The inner location data fetched from the service
+     * 	@private
+     */
+    formatInnerLocationData: function(eventInnerLocationData) {
+        if (eventInnerLocationData !== null && eventInnerLocationData !== undefined && eventInnerLocationData.length > 0) {
```
Comment (vijaysamanthapudi) [inline KonyAccelerate/controllers/mobile/frmLoginController/frmLoginController.js:57]: Use kony.sdk.isNullOrUndefined

File: KonyAccelerate/controllers/mobile/frmLoginController/frmLoginController.js [hunk 3]
```diff
@@ -1,6 +1,74 @@
...
+     *	@function fetchEventData
+     * 	@description This function is used to fetch all the event data required
+     * 	@private
+     */
+    fetchEventData: function() {
+        var objectServiceName = eventConstants.OBJECT_SERVICE_NAME;
+        var objectName = eventConstants.EVENT_OBJECT_NAME;
+        var queryParams = {
+            "$filter": "((SoftDeleteFlag ne true) or (SoftDeleteFlag eq null))",
+            "$expand": "event_inner_location,wifi_info,location"
```
Comment (vijaysamanthapudi) [inline KonyAccelerate/controllers/mobile/frmLoginController/frmLoginController.js:13]: if possible try to move these values to  event_inner_location , wifi_info, location to constants file

File: KonyAccelerate/userwidgets/com.konymp.linkedinlogin/modules/require/linkedinloginController.js [hunk 4]
```diff
@@ -199,7 +199,7 @@ define(['com/konymp/linkedinlogin/constants'],function(constantsLocal) {
          * @function failureInFetchingEmailId
          * @description Error Callback function 
          * @param {Object} error - gives the error status
-         * @private
+         * @privateob
```
Comment (vigneshwarankarunanithi) [inline KonyAccelerate/userwidgets/com.konymp.linkedinlogin/modules/require/linkedinloginController.js]: change the spelling, it should be 'private'

**Review by vigneshwarankarunanithi — APPROVED** [review]

File: KonyAccelerate/controllers/mobile/frmEventLocationController/frmEventLocationController.js (hunk 1 above)
Comment (krishk95) [reply KonyAccelerate/controllers/mobile/frmEventLocationController/frmEventLocationController.js]: Addressed

**Pull request merged**